harness/determined/common/experimental/_util.py

```diff
@@ -0,0 +1,18 @@
+import enum
+
+from determined.common.api import bindings
+
+
+class OrderBy(enum.Enum):
+    """
```

Comment:
nit: small request to follow [style guide format](https://google.github.io/styleguide/pyguide.html#384-classes) for this docstring.